iceoryx_hoofs/source/posix_wrapper/named_pipe.cpp

```diff
@@ -178,9 +217,47 @@ cxx::expected<bool, IpcChannelError> NamedPipe::unlinkIfExists(const IpcChannelN
     return cxx::error<IpcChannelError>(IpcChannelError::INTERNAL_LOGIC_ERROR);
 }
 
+cxx::expected<IpcChannelError> NamedPipe::trySend(const std::string& message) const noexcept
```

Comment:
`trySend` and `tryReceive` are not tested. Since the ipc channel will probably be refactored it doesn't make sense to add the tests now, but can you please add a todo in some related issue?

Reply:
I added it to issue #832